en/9-address-security-privacy-risks.md

```diff
@@ -68,69 +93,53 @@ altLangPage: 9-gerer-risques-matiere-securite-protection-renseignements-personne
+Embedding privacy protection in the design of digital applications or open data increases political legitimacy and public confidence, and privacy safeguards are a necessary condition for a successful shift to a digital Government of Canada. Digital services also have the potential to enhance privacy rights, for example, by facilitating access to and correction of personal information.
 
- **(Digital Service Standard (Ontario))**
+Organizations have a responsibility to ensure that the data under their care remains protected at all times, including in the process of sharing with external partners and within their own network. This requires an understanding what data is worth protecting, manage who and what can access it, and build effective defenses that both support innovation and protect the investment made in services and associated assets.
 
 </div>
 
 <section class="dpgn-section-checklist">
 
 ### Checklist
 
-**[TODO: Add/revise checklist items]**
-
-<section class="dpgn-section-stage dpgn-stage-alpha">
-
-#### Alpha stage
-
-- determine what data is being collected (name, address, postal code, etc.) **(Digital Service Standard (Ontario))**
-- determine how the data is being transmitted **(Digital Service Standard (Ontario))**
-- determine where and how the data is being stored **(Digital Service Standard (Ontario))**
-- determine how the data will be used **(Digital Service Standard (Ontario))**
-- identify security and privacy threats **(Digital Service Standard (Ontario))**
-- document interactions with business and information risk teams (for example Corporate Security and Information, Privacy and Archives) **(Digital Service Standard (Ontario))**
-- determine how privacy and security regulations will be met without putting delivery at risk **(Digital Service Standard (Ontario))**
-- document any outstanding legal concerns, such as data protection or data sharing **(Digital Service Standard (Ontario))**
-- document the privacy policy and rationale **(Digital Service Standard (Ontario))**
+* Ensure that a preliminary privacy impact assessment (PPIA) or privacy impact assessment (PIA) has been completed if personal information is involved in the service.
+* Determine, in consultation with a records officer, what data is collected and why, how it is used or shared, how it is stored and secured, and how long it is kept.
+* Ensure that digital solutions and open data initiatives comply with requirements regarding the collection and protection of personal information.
+* Where collecting personal information, inform users about privacy rights and protections, and about their right to access and correct their own personal information.
+* Use appropriate de-identification strategies to minimize the risk of disclosing personal information.
+* Incorporate privacy safeguards into partnership and data sharing agreements.
+* Develop your system with the 7 principles of Privacy by Design in mind.

```

Comment:
This should link to the 7 principles of Privacy by Design.

Reply:
Recommend using this reference - https://www.ipc.on.ca/wp-content/uploads/Resources/7foundationalprinciples.pdf

Reply:
Turned into a link using the recommended reference.